doc/source/extending-jupyterhub.rst

```diff
@@ -44,31 +45,60 @@ To instruct JupyterHub to use this image, simply add this to your ``config.yaml`
 
 You can then `apply the change <#applying-configuration-changes>`_ to the config as usual.
 
+
 Setting memory and CPU guarantees / limits for your users
 ---------------------------------------------------------
 
-Each user on your JupyterHub gets a slice of memory and CPU to use. By default, each user is
-*guaranteed* 1G of RAM, but no enforced limits. This means each user is guaranteed at minimum
-1G of RAM, but no exact upper limit (this will vary based on various factors outside of your
-control). We recommend most people change this!
+Each user on your JupyterHub gets a slice of memory and CPU to use. There are
+two ways in which you can provision these resources: as *guaranteed* resources,
+or as *limited* resources.
+
+A *guaranteed* resource means that all users will have *at least* this resource
+available at all times, but they may be able to use more resource if they're
+available. For example, if users are *guaranteed* 1G of RAM, users can technically
+use more than 1G of RAM if these resources aren't being used by other computers.
 
-If you want all your users to have guaranteed access to 1Gi of RAM, and no more, you can add the
-following to your config.yaml and do an upgrade:
+A *limited* resource sets a hard limit on the resources available. In the example
```

Comment:
~~In the example above,~~ If a memory is set up as a *limited* resource with a 1G memory limit, an individual user could use nor more than 1G of memory (RAM), ...